src/mongocxx/test/database.cpp

```diff
@@ -42,6 +42,12 @@ using bsoncxx::builder::basic::make_document;
 
 bool check_for_collections(cursor cursor, std::set<std::string> expected_colls) {
     for (auto&& coll : cursor) {
+        // Skip system collections
+        auto pos = coll["name"].get_utf8().value.to_string().find("system.");
```

Comment:
This exact subexpression is reiterated on line 51. Maybe pull it into a variable?

Reply:
Fixed in a separate commit which I'll squash before merge.